version.gradle.kts

```diff
@@ -1,5 +1,5 @@
 // do NOT update that variable manually - it is managed by the pre/post release scripts
-val distroVersion = "2.0.0-alpha"
+val distroVersion = "2.1.0-SNAPSHOT"
```

Comment:
```suggestion
val distroVersion = "2.0.1-SNAPSHOT"
```

Reply:
I think it should be `alpha-SNAPSHOT`. Going with `2.1.0` would let us be in sync with upstream, do we want that?

Reply:
I mean, I guess that's fine. The problem has been that people then expect them to be in lockstep and when they diverge they freak out. :)

Reply:
imo, if possible, we should keep them in lockstep so that upstream major.minor.0 release would correspond to our major.minor.0, patch releases may differ.